evap/staff/templates/staff_user_merge_course_list.html

```diff
@@ -0,0 +1,14 @@
+{% for group in content_list %}
```

Comment:
As this file is not only used for courses, but also for contributions and participations and maybe for any other grouped list in the future, I suggest renaming it to `staff_user_merge_grouped_list.html`, and replacing `content_list` with `grouped_list`. What do you think?